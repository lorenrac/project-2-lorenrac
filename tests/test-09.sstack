// IF with scoped body.
IF ("1") {
  PRINT "First line of IF body!"
  PRINT "Second line of IF body!"
}
IF ("") {
  PRINT "This shouldn't print!"
  PRINT "Nor should this!"
}
