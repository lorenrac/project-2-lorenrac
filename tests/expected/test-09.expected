First line of IF body!
Second line of IF body!
